syntax = "proto3";

package controller;

option go_package = "../gen";

service ServiceSum {
    rpc
}

service StringService {
  rpc SendString (StringRequest) returns (StringReply) {};
}

message StringRequest {
  string message = 1;
}

message StringReply {
  string message = 1;
}
